Sources/Core/Web3Error/Web3Error.swift

```diff
@@ -25,7 +25,7 @@ public enum Web3Error: Error {
     case generalError(err: Error)
     case unknownError
 
-    public var errorDescription: String {
+    public var errorDescription: String? {
```

Comment:
Changed type to comply with `LocalizedError` protocol and enable `localizedDescription` return value from `errorDescription`.

Reply:
Thou it's a bit confusing for me when I've got error messages on my language in console. Also it's usually awfully translated. But anyway, this is must.